src/fswAlgorithms/transDetermination/ephemNavConverter/ephemNavConverter.cpp

```diff
@@ -20,35 +20,32 @@
 #include "fswAlgorithms/transDetermination/ephemNavConverter/ephemNavConverter.h"
 #include "architecture/utilities/linearAlgebra.h"
 
-/*! This resets the module to original states.
+/*! Reset method for the BSK module adapter interface. This method also calls the algorithm reset method.
  @return void
  @param callTime The clock time at which the function was called (nanoseconds)
  */
-void EphemNavConverter::reset(uint64_t callTime)
-{
+void EphemNavConverter::reset(uint64_t callTime) {
     // check if the required message has not been connected
     if (!this->ephInMsg.isLinked()) {
         this->bskLogger.bskLog(BSK_ERROR, "Error: ephemNavConverter.ephInMsg wasn't connected.");
     }
+
+    // Call the algorithm reset method
+    this->algorithm.reset(callTime);
 }
 
-/*! This method reads in the ephemeris messages and copies the translation
-    ephemeris to the navigation translation interface message.
+/*! Update method for the BSK module adapter interface. This method also calls the algorithm update method.
  @return void
- @param callTime The clock time at which the function was called (nanoseconds)
+ @param callTime [ns] Time the method is called
  */
-void EphemNavConverter::updateState(uint64_t callTime)
-{
-    NavTransMsgPayload tmpOutputState = {};
-
-    /*! - read input ephemeris message */
-    EphemerisMsgPayload tmpEphemeris  = this->ephInMsg();
+void EphemNavConverter::updateState(uint64_t callTime) {
+    auto localEphemInMsg = EphemerisMsgPayload();
```

Comment:
:small_red_triangle_down: This is a msg payload not the msg handle. We should probably name it as such.